wrappers/python/tests/layer/test_llama_attention.py

```diff
@@ -79,48 +117,59 @@ def generate_inputs(params: LlamaAttentionTestParams):
     x_grad = x_type(x_traits, x_distr, 0)
     X = TensorMoments(x_value, x_grad, grad_required=True)
     x_random = np.random.randn(*x_shape)
-    x_nntile = np.array(x_random, dtype=params.dtype, order='F')
+    x_nntile = np.array(x_random, dtype=params.dtype, order="F")
     x_value.from_array(x_nntile)
     x_torch = torch.Tensor(x_nntile.T)
-    nntile_layer = nntile.layer.LlamaAttention.from_torch(torch_layer, X, \
-            params.n_head_tile)
+    nntile_layer = nntile.layer.LlamaAttention.from_torch(
+        torch_layer, X, params.n_head_tile
+    )
     y_grad_random = np.random.randn(*x_shape)
-    y_grad_nntile = np.array(y_grad_random, dtype=params.dtype, order='F')
+    y_grad_nntile = np.array(y_grad_random, dtype=params.dtype, order="F")
     nntile_layer.y.grad.from_array(y_grad_nntile)
     y_grad_torch = torch.Tensor(y_grad_nntile.T)
     return torch_layer, nntile_layer, x_torch, y_grad_torch
 
+
 @pytest.mark.parametrize("params", TEST_PARAMS)
 class TestLlamaAttention:
-    def test_from_to_torch(self, starpu_simple, torch_rng, \
-            params: LlamaAttentionTestParams):
+    def test_from_to_torch(
+        self, starpu_simple, torch_rng, params: LlamaAttentionTestParams
+    ):
```

Comment:
I didn't get what `from_to_torch` mean. Does it test that layers are identically initialized, doesn't it?

Reply:
It means testing `from_torch` and `to_torch` functionality. Does `test_from_torch_and_to_torch` sound better?

Reply:
I would name it `test_torch_coercion` or `test_torch_interop` or `test_torch_casting`. Something like that.

Reply:
`test_torch_casting` sounds good